### PR TITLE
Fix part of #2003: Replacing space_24dp

### PR DESCRIPTION
**Pull request by Arjupta**
## Explanation

Fixes part of #2003 
This PR replaces the use of space_24dp with the dimensions having contextful names.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by rt4914** [comment]
@ARJUPTA PTAL at tests.

**Comment by Arjupta** [comment]
@rt4914 i am bit unable to understand these logs

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -60,6 +59,75 @@
...
+  <dimen name="drag_drop_item_exploration_view_margin_end">24dp</dimen>
+  <dimen name="drag_drop_item_question_view_margin_top">24dp</dimen>
+  <dimen name="feedback_item_exploration_split_view_margin_start">24dp</dimen>
+  <dimen name="feedback_item_exploration_split_view_margin_top">24dp</dimen>
+  <dimen name="feedback_item_exploration_view_margin_start">24dp</dimen>
+  <dimen name="feedback_item_exploration_view_margin_top">24dp</dimen>
+  <dimen name="feedback_item_question_view_margin_top">24dp</dimen>
+  <dimen name="fraction_interaction_item_conversation_view_margin_end">24dp</dimen>
+  <dimen name="next_button_exploration_view_padding_end">24dp</dimen>
+  <dimen name="next_button_question_view_padding_end">24p</dimen>
```
Comment (rt4914) [inline app/src/main/res/values/dimens.xml]: Nit: `24dp`

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -60,6 +59,75 @@
...
+  <dimen name="next_button_exploration_view_padding_start">24dp</dimen>
+  <dimen name="next_button_question_view_padding_start">24p</dimen>
```
Comment (rt4914) [inline app/src/main/res/values/dimens.xml]: Nit: `24dp`

Check elsewhere too.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Reason of test fails.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**